src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -0,0 +1,125 @@
+package org.openmrs.maven.plugins;
+
+import org.apache.maven.execution.MavenSession;
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.BuildPluginManager;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.components.interactivity.Prompter;
+import org.openmrs.maven.plugins.model.Artifact;
+import org.openmrs.maven.plugins.model.Server;
+import org.openmrs.maven.plugins.utility.ConfigurationManager;
+import org.openmrs.maven.plugins.utility.SDKConstants;
+
+import java.util.List;
+
+/**
+ *
+ * @goal setup
+ * @requiresProject false
+ *
+ */
+public class Setup extends AbstractMojo {
```

Comment:
I wonder if extending SetupPlatform would work. That is to eliminate the need to declare fields twice. You could try that.